### DISCUSSION TITLE
Battery powered device availability suggestions?

### DISCUSSION THREAD

**Discussion by mikerenna** [Koenkk/zigbee2mqtt#13602] [Devices]
Does anyone have any tips on improving performance of battery powered devices as it relates to availability? I've found that regardless of type/manufacturer, battery devices periodically become unavailable. It does not appear to be a function of battery levels. I've been looking at availability reporting to see if there is a configuration suggestion that mitigates this but it hasn't been clear if there is a best practice there. Any thoughts or suggestions? Am I alone in this observation? 

Thanks in advance. 
-Mike

**Comment by Koenkk** [comment]
You can increase the availability timeout for this device to e.g. 50 hours (https://www.zigbee2mqtt.io/guide/configuration/device-availability.html#availability-advanced-configuration)

**Comment by mikerenna** [comment]
Thanks I'll give that a try. Is the underlying issue that these devices don't check in within 24 hours?
________________________________
